shared/constants/network.ts

```diff
@@ -538,7 +678,106 @@ export const BUYABLE_CHAINS_MAP: {
       srn: 'ethereum',
       currencyCode: CURRENCY_SYMBOLS.ETH,
     },
-    coinbasePayCurrencies: [CURRENCY_SYMBOLS.ETH, 'USDC', 'DAI'],
+    coinbasePayCurrencies: [
+      CURRENCY_SYMBOLS.ETH,
+      '1INCH',
+      'AAVE',
+      'ABT',
+      'ACH',
+      'AGLD',
+      'AMP',
+      'ANKR',
+      'APE',
+      'ARPA',
+      'ASM',
+      'AUCTION',
+      'AXS',
+      'BADGER',
+      'BAL',
+      'BAND',
+      'BAT',
+      'BNT',
+      'BOBA',
+      'BOND',
+      'BTRST',
+      'CHZ',
+      'CLV',
+      'COMP',
+      'COTI',
+      'CRO',
+      'CRV',
+      'CTSI',
+      'CVC',
+      'DAI',
+      'DDX',
+      'DNT',
+      'ENJ',
+      'ENS',
+      'FARM',
+      'FET',
+      'FORTH',
+      'FX',
+      'GNO',
+      'GRT',
+      'GTC',
+      'IOTX',
+      'JASMY',
+      'KEEP',
+      'KNC',
+      'KRL',
+      'LCX',
+      'LINK',
+      'LPT',
+      'LRC',
+      'MANA',
+      'MASK',
+      'MATIC',
+      'MIR',
+      'MKR',
+      'MLN',
+      'MTL',
+      'NKN',
+      'NMR',
+      'NU',
+      'OGN',
+      'OMG',
+      'OXT',
+      'PAX',
+      'PERP',
+      'PLA',
+      'POLY',
+      'QNT',
+      'QUICK',
+      'RAD',
+      'RAI',
+      'RARI',
+      'REN',
+      'REP',
+      'REQ',
+      'RLC',
+      'RLY',
+      'SAND',
+      'SHIB',
+      'SKL',
+      'SNX',
+      'STORJ',
+      'SUKU',
+      'SUSHI',
+      'SWFTC',
+      'TRAC',
+      'TRB',
+      'TRIBE',
+      'TRU',
+      'UMA',
+      'UNI',
+      'USDC',
+      'USDT',
+      'WBTC',
+      'WCFG',
+      'XYO',
+      'YFII',
+      'ZRX',
```

Comment:
Have to use the CURRENCY_SYMBOLS constant so that types aren't widened to strings.

Reply:
@brad-decker change ready, didnt do it that way initially since i choose to follow the current format where just the native currency was a constant